Xamarin.Essentials/Types/PlatformExtensions/TimeSpanExtensions.shared.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Xamarin.Essentials
+{
+    public static partial class TimeSpanExtensions
+    {
+        public static TimeSpan RoundToNearestMinutes(this TimeSpan input, int minutes)
```

Comment:
Where are we using this - the name doesn't seem to indicate how to call it.

Reply:
Hmm this function is currently only being used twice in CalendarEventAddViewModel.cs

e.g.
TimeSpan startTime = DateTime.Now.AddHours(1).TimeOfDay.RoundToNearestMinutes(30);

Perhaps I should just move that function into CalendarEventAddViewModel since it is not being used anywhere else currently?

Reply:
Yeh if this is only used by the sample app, we should remove from the core library